File: neetcode/arrays-hashing/two_sum.go
package arrayshashing

func twoSum(nums []int, target int) []int {
	result := []int{}
	for i, v := range nums {
		for j, w := range nums {
			if i != j {
				sum := v + w
				if sum == target {
					result = append(result, i, j)
					return result
				}
			}
		}
	}
	return result
}

File: leetcode/1-two-sum/two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        numsDict: dict[int, int] = {}
        for i, v in enumerate(nums):
            more = target - v
            if more in numsDict:
                return [numsDict[more], i]
            numsDict[v] = i
        return []
